pkgs/by-name/op/opa-envoy-plugin/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+
+  enableWasmEval ? false,
+}:
+
+assert
+  enableWasmEval && stdenv.hostPlatform.isDarwin
```

Comment:
Any reason not to put this conditional in meta.broken instead of a separate assert?

Reply:
I think the assert will produce an output on the command line?

Reply:
As this project is similar in many ways to OPA, this was copied from the open-policy-agent package, see here: https://github.com/NixOS/nixpkgs/blame/master/pkgs/by-name/op/open-policy-agent/package.nix#L11-L14. 

Let me know if there is a preferred way to do such things now and I can get it updated.